docs/src/modules/components/DemoToolbar.js

```diff
@@ -226,7 +226,7 @@ export default function DemoToolbar(props) {
   };
 
   const handleCodeLanguageClick = (event, clickedCodeVariant) => {
-    if (codeVariant !== clickedCodeVariant) {
+    if (clickedCodeVariant !== null && codeVariant !== clickedCodeVariant) {
```

Comment:
**Off-topic**

I wonder if a toggle button should no enforce to retain the active value when set to `exclusive`. This demos feels weird https://next.material-ui.com/components/toggle-button/#exclusive-selection, what does no text alignment mean? 😁

Reply:
> Should ToggleButtonGroup enforce to retain the active value when set to `exclusive`? 

I think the confusion comes from the ambiguity of `exclusive`. With your proposal exactly one value would have to be selected. The current implementation allows 0 or 1 values to be selected. In other words: With your proposal we could not de-select.

But this doesn't block this PR so I'm just assuming this means approval. Please approve explicitly in the future and generally be more specific in your reviews. They require almost always another round-trip that could've been avoided by explicit messaging.

Reply:
> But this doesn't block this PR so I'm just assuming this means approval.

@eps1lon You are right.

We were facing a similar pain in the X repo. A couple of weeks ago, in the retrospective, we settled on using [**Off-topic**](https://www.notion.so/mui-org/Engineering-Dojo-Kun-ec1615511f594782a8a2adeec3e25143) when we wanted to start discussions on parallel problems that aren't relevant for merging the PR. Sometimes, it's about potentially going after the root cause, others about weird stuff in the code that was already present, or else.